apps/api/.env.example

```
@@ -38,3 +38,12 @@ MODEL_CONTEXT_WINDOW_TOKENS=
 # model. Takes precedence over the MODEL_CONTEXT_WINDOW_TOKENS derivation. Usually
 # leave empty; the eval suite sets it very low to exercise compaction cheaply.
 COMPACTION_TOKEN_THRESHOLD=
+
+# Run liveness (#48). A per-run deadman job fires RUN_TIMEOUT_SECONDS
+# (default 300) after enqueue: a terminal run is left alone, a run with a fresh
+# heartbeat is re-checked later, and a run whose heartbeat is older than
+# RUN_HEARTBEAT_STALE_SECONDS (default 60) is expired. The executing worker
+# stamps the heartbeat every RUN_HEARTBEAT_SECONDS (default 15).
+RUN_TIMEOUT_SECONDS=
+RUN_HEARTBEAT_STALE_SECONDS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUN_HEARTBEAT_STALE_SECONDS key should go before the RUN_TIMEOUT_SECONDS key

(UnorderedKey)